diff --git a/node_modules/notion-to-md/build/notion-to-md.js b/node_modules/notion-to-md/build/notion-to-md.js
index 7b3ae32..c26a71a 100644
--- a/node_modules/notion-to-md/build/notion-to-md.js
+++ b/node_modules/notion-to-md/build/notion-to-md.js
@@ -151,6 +151,7 @@ ${md.addTabSpace(mdBlocks.parent, nestingLevel)}
             case "bookmark":
             case "embed":
             case "link_preview":
+            case "link_to_page":
                 {
                     let blockContent;
                     if (type === "bookmark")
@@ -159,6 +160,8 @@ ${md.addTabSpace(mdBlocks.parent, nestingLevel)}
                         blockContent = block.embed;
                     if (type === "link_preview")
                         blockContent = block.link_preview;
+                    if (type === "link_to_page" && block.link_to_page.type === "page_id")
+                        blockContent = { url: "/"+block.link_to_page.page_id };
                     if (blockContent)
                         return md.link(type, blockContent.url);
                 }
